packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol

```diff
@@ -90,6 +92,11 @@ abstract contract SpokeConnector is Connector, ConnectorManager, WatcherClient,
 
   // ============ Public Storage ============
 
+  /**
+   * @notice Duration of the snapshot
+   */
+  uint256 public constant SNAPSHOT_DURATION = 30 minutes;
```

Comment:
see above comments re: time (one thing that makes this annoying is it would have to be configured per-connector, so instantiated with the constructors)

Reply:
Sorry what comment are you referring to?

Reply:
woah - odd. looks like I deleted the wrong comment

the question is what about the duration being in blocks -- doesn't actually work with the snapshot duration since that's used to generate the ID, but we generally use blocks for dispute timers. we have seen chains halt, which is problematic if you are measuring dispute windows in time since the dispute window will elapse without you ever actually having the opportunity to submit.

Reply:
This is related to the comment about changing the DISPUTE_TIME to blocks so its already addressed.